plugin/evm/mempool_atomic_gossiping_test.go

```diff
@@ -108,13 +108,15 @@ func TestMempoolMaxMempoolSizeHandling(t *testing.T) {
 	// shortcut to simulated almost filled mempool
 	mempool.maxSize = 0
 
-	assert.ErrorIs(mempool.AddTx(tx), errTooManyAtomicTx)
+	err := mempool.AddTx(tx)
```

Comment:
could we revert this file to `master` entirely? all the changes are only syntactical.